src/e2e-test/features/gcsmove/GCSMove.feature

```diff
@@ -0,0 +1,79 @@
+@GCSMove
+Feature:GCSMove - Verification of successful objects move from one bucket to another
+
+  @CMEK @GCS_CSV_TEST @GCS_SINK_TEST
+  Scenario:Validate successful move object from one bucket to another new bucket
+    Given Open Datafusion Project to configure pipeline
+    When Expand Plugin group in the LHS plugins list: "Conditions and Actions"
+    When Select plugin: "GCS Move" from the plugins list as: "Action"
+    When Navigate to the properties page of plugin: "GCS Move"
+    Then Enter GCSMove property projectId "projectId"
+    Then Enter GCSMove property source path "gcsCsvFile"
+    Then Enter GCSMove property destination path
+    Then Enter GCSMove property encryption key name "cmekGCS" if cmek is enabled
+    Then Validate "GCSMove" plugin properties
+    Then Close GCSMove properties
+    Then Save and Deploy Pipeline
+    Then Run the Pipeline in Runtime
+    Then Wait till pipeline is in running state
+    Then Open and capture logs
+    Then Verify the pipeline status is "Succeeded"
+    Then Validate GCSMove successfully moved object "gcsCsvFile" to destination bucket
+    Then Validate the cmek key "cmekGCS" of target GCS bucket if cmek is enabled
+
+  @GCS_READ_RECURSIVE_TEST @GCS_SINK_TEST
+  Scenario:Validate successful move objects from one bucket to another with Move All Subdirectories set to true
+    Given Open Datafusion Project to configure pipeline
+    When Expand Plugin group in the LHS plugins list: "Conditions and Actions"
+    When Select plugin: "GCS Move" from the plugins list as: "Action"
+    When Navigate to the properties page of plugin: "GCS Move"
+    Then Enter GCSMove property projectId "projectId"
+    Then Enter GCSMove property source path "gcsMoveReadRecursivePath"
+    Then Enter GCSMove property destination path
+    Then Select GCSMove property move all subdirectories as "true"
```

Comment:
Test with more than 1 subdirectory. Also test move all subdirectories as "false"

Reply:
test move all subdirectories as "false"
Added the test

Reply:
Add test for auto create location set to non-default value
Added the test